app/assets/stylesheets/nfg_ui/network_for_good/admin/application.scss

```diff
@@ -13,7 +13,6 @@
 
 // NFG Theme
 @import 'nfg_theme/*';
-@import 'nfg_theme/custom/email_preview'; // TODO: Figure out why this isn't loading by the import nfg/theme/*
```

Comment:
Were you able to figure out how to get this loading into the `@import 'nfg_theme/*';`? I was wondering if it was the nested  `/custom/` folder (e.g.: `nfg_theme/**/*`, maybe?))

Reply:
I don't have an exact answer for why it wasn't working in the first place, but it's pulling it without that line. So magically working now?! Could you double check by pulling this branch locally to see if the styles in that folder are in the compiled stylesheet? It was for me after I checked and rechecked. Thanks for pointing this out.